spec/lib/common/system-uuid-spec.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2016, EMC, Inc.
+'use strict';
+
+describe("SystemUuid", function() {
+    var subject;
+    var regEx = /^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$/;
```

Comment:
This regex is not the same with https://github.com/RackHD/on-core/blob/master/lib/common/constants.js#L197